client/src/main/java/io/pravega/client/stream/impl/Credentials.java

```diff
@@ -24,8 +22,8 @@
     String getAuthenticationType();
 
     /**
-     * Returns authorization parameters used by this specific authentication type.
-     * @return The map of authentication headers and values.
+     * Returns the authorization token to be sent to Pravega.
+     * @return A token in token68-compatible format (as defined by RFC 7235).
```

Comment:
One clarification: what is specific to `token68-compatible` and do we need a precondition in the implementation to make sure that we have the right format? For example, in this line:

https://github.com/pravega/pravega/pull/2385/files#diff-4f1465428bd2115aec963494bd80927bR26

The encoding does seem to have anything special for the tolken.

Reply:
I am simply restating the HTTP Authentication spec (RFC 7235) which outlines the general use of the `Authorization` header; the value is largely opaque (to be interpreted by the plugin).    I'll rephrase as 'a token value as specified in RFC-7235'.